// ngx-loaders-css/esm2020/lib/ngx-loaders-css.module.mjs
import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';
import { NgxLoadersCssComponent } from './ngx-loaders-css.component';
import * as i0 from "@angular/core";
export class NgxLoadersCssModule {
}
NgxLoadersCssModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.0", ngImport: i0, type: NgxLoadersCssModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
NgxLoadersCssModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.0.0", ngImport: i0, type: NgxLoadersCssModule, declarations: [NgxLoadersCssComponent], imports: [CommonModule], exports: [NgxLoadersCssComponent] });
NgxLoadersCssModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.0.0", ngImport: i0, type: NgxLoadersCssModule, imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.0", ngImport: i0, type: NgxLoadersCssModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [NgxLoadersCssComponent],
                    exports: [NgxLoadersCssComponent],
                }]
        }] });
